flash-messages/Gemfile

```diff
@@ -0,0 +1,20 @@
+source "https://rubygems.org"
+
+gem "sinatra"
+gem "data_mapper"
+gem "rack-flash3"
```

Comment:
I'd rather we didn't use a gem for this, instead we could use a helper method to demonstrate a read-once hash-retrieval.

```
def get_flash(key)
  session[:flash].delete(:key)
end
def set_flash(key, value)
  session[:flash] = value
end
```